python-sdk/src/astro/databases/base.py

```diff
@@ -345,7 +345,10 @@ def drop_table(self, table: BaseTable) -> None:
         :param table: The table to be deleted.
         """
         statement = self._drop_table_statement.format(self.get_table_qualified_name(table))
-        self.run_sql(statement)
+        if self.sql_type == "mssql":
```

Comment:
@phanikumv Why do we need this?

Reply:
Also, if this is needed we should move it to databases/mssql.py by overriding `drop_table` method.

Reply:
we need this to be able to add autocommit flag for certain sql commands. Will move it to `databases/mssql.py`